<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OnlineMenuScreen</class>
 <widget class="QWidget" name="OnlineMenuScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="tTitleLabel" name="label">
     <property name="text">
      <string>Online Play</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>391</width>
        <height>536</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="FocusBarrier" name="barrierTop">
         <property name="text">
          <string>^</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCommandLinkButton" name="openLobbyButton">
         <property name="text">
          <string>Create Challenge</string>
         </property>
         <property name="description">
          <string>Challenge a friend to a match</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCommandLinkButton" name="joinLobbyButton">
         <property name="text">
          <string>Accept Challenge</string>
         </property>
         <property name="description">
          <string>Accept a friend's challenge</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCommandLinkButton" name="relationsButton">
         <property name="text">
          <string>Friends</string>
         </property>
         <property name="description">
          <string>Add or remove friends and blocked users</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCommandLinkButton" name="accountButton">
         <property name="text">
          <string>Account</string>
         </property>
         <property name="description">
          <string>Manage your account</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCommandLinkButton" name="exitButton">
         <property name="text">
          <string>Main menu</string>
         </property>
         <property name="description">
          <string>Disconnect from the server</string>
         </property>
         <property name="type" stdset="0">
          <string>destructive</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="FocusBarrier" name="barrierBottom">
         <property name="text">
          <string>v</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>138</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="GamepadHud" name="gamePadControls" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GamepadHud</class>
   <extends>QWidget</extends>
   <header location="global">gamepadhud.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FocusBarrier</class>
   <extends>QPushButton</extends>
   <header location="global">focusbarrier.h</header>
  </customwidget>
  <customwidget>
   <class>tTitleLabel</class>
   <extends>QLabel</extends>
   <header location="global">ttitlelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
